CONTRIBUTING.rst

```diff
@@ -42,22 +42,37 @@ To update the pypy snapshot (and thus our python dependencies), run
 tahoe-lafs
 ..........
 
-We depend on pinned commit of tahoe-lafs.
-To update to the latest commit, run
+We test against a pinned commit of Tahoe-LAFS master.
+To update to the current master@HEAD revision, run:
 
 .. code:: shell
 
-   nix-shell --run 'niv update tahoe-lafs --branch master'
+   nix-shell --run 'niv update tahoe-lafs-master --branch master'
+
+We intend for these updates to be performed periodically.
+At the moment, they must be performed manually.
+It might be worthwhile to `automate this process <https://github.com/PrivateStorageio/ZKAPAuthorizer/issues/287>` in the future.
 
 It is also possible to pass ``pull/<pr-number>/head`` to test against a specific PR.
 
+The Nix package depends on a version of Tahoe-LAFS determined by the "tahoe-lafs" niv entry.
+When feasible this is a released version of Tahoe-LAFS.
+To update to a new release, run:
+
+.. code:: shell
+
+   nix-shell --run 'niv update --rev tahoe-lafs-A.B.C tahoe-lafs'
+
+When it is not feasible to use a released version of Tahoe-LAFS,
+niv's ``--branch`` or ``--rev`` features can be used to update this dependency.
+
 .. note::
 
    Since tahoe-lafs doesn't have correct version information when installed from a github archive,
    the packaging in ``default.nix`` includes a fake version number.
    This will need to be update manually at least when the minor version of tahoe-lafs changes.
 
-If you want to test multiple versions, you can add an additional source, pointing at other version
+If you want to test additional versions, you can add an additional source, pointing at other version.
```

Comment:
I think I'd be inclined to put the `tahoe-lafs-master` instructions after this, partly as an example of testing against additional versions.